fbs/request.fbs

```diff
@@ -0,0 +1,86 @@
+include "router.fbs";
+include "transport.fbs";
+include "worker.fbs";
+
+namespace FBS.Request;
+
+enum Method: uint8 {
+  WORKER_CLOSE = 0,
+  WORKER_DUMP,
+  WORKER_GET_RESOURCE_USAGE,
+  WORKER_UPDATE_SETTINGS,
+  WORKER_CREATE_WEBRTC_SERVER,
+  WORKER_CREATE_ROUTER,
+  WORKER_WEBRTC_SERVER_CLOSE,
+  WEBRTC_SERVER_DUMP,
+  WORKER_CLOSE_ROUTER,
+  ROUTER_DUMP,
+  ROUTER_CREATE_WEBRTC_TRANSPORT,
+  ROUTER_CREATE_WEBRTC_TRANSPORT_WITH_SERVER,
+  ROUTER_CREATE_PLAIN_TRANSPORT,
+  ROUTER_CREATE_PIPE_TRANSPORT,
+  ROUTER_CREATE_DIRECT_TRANSPORT,
+  ROUTER_CLOSE_TRANSPORT,
+  ROUTER_CREATE_ACTIVE_SPEAKER_OBSERVER,
+  ROUTER_CREATE_AUDIO_LEVEL_OBSERVER,
+  ROUTER_CLOSE_RTP_OBSERVER,
+  TRANSPORT_DUMP,
+  TRANSPORT_GET_STATS,
+  TRANSPORT_CONNECT,
+  TRANSPORT_SET_MAX_INCOMING_BITRATE,
+  TRANSPORT_SET_MAX_OUTGOING_BITRATE,
+  TRANSPORT_RESTART_ICE,
+  TRANSPORT_PRODUCE,
+  TRANSPORT_PRODUCE_DATA,
+  TRANSPORT_CONSUME,
+  TRANSPORT_CONSUME_DATA,
+  TRANSPORT_ENABLE_TRACE_EVENT,
+  TRANSPORT_CLOSE_PRODUCER,
+  TRANSPORT_CLOSE_CONSUMER,
+  TRANSPORT_CLOSE_DATA_PRODUCER,
+  TRANSPORT_CLOSE_DATA_CONSUMER,
+  PRODUCER_DUMP,
+  PRODUCER_GET_STATS,
+  PRODUCER_PAUSE,
+  PRODUCER_RESUME,
+  PRODUCER_ENABLE_TRACE_EVENT,
+  CONSUMER_DUMP,
+  CONSUMER_GET_STATS,
+  CONSUMER_PAUSE,
+  CONSUMER_RESUME,
+  CONSUMER_SET_PREFERRED_LAYERS,
+  CONSUMER_SET_PRIORITY,
+  CONSUMER_REQUEST_KEY_FRAME,
+  CONSUMER_ENABLE_TRACE_EVENT,
+  DATA_PRODUCER_DUMP,
+  DATA_PRODUCER_GET_STATS,
+  DATA_CONSUMER_DUMP,
+  DATA_CONSUMER_GET_STATS,
+  DATA_CONSUMER_GET_BUFFERED_AMOUNT,
+  DATA_CONSUMER_SET_BUFFERED_AMOUNT_LOW_THRESHOLD,
+  RTP_OBSERVER_PAUSE,
+  RTP_OBSERVER_RESUME,
+  RTP_OBSERVER_ADD_PRODUCER,
+  RTP_OBSERVER_REMOVE_PRODUCER,
+}
+
+union Body {
+  FBS.Router.CreateWebRtcTransportRequest,
+  FBS.Router.CreatePlainTransportRequest,
+  FBS.Router.CreatePipeTransportRequest,
+  FBS.Transport.ConsumeRequest,
+  FBS.Worker.UpdateableSettings,
```

Comment:
Why UpdateableSettings instead of UpdateSettingsRequest?

IMHO all entries here should have the method name (literal) plus ".Request".

Or another more consistent and easier proposal:

"FBS.Worker.UpdateSettings.Request"

But still we are not indicating here that this is about the BODY of the request... so not sure if I like it.

Reply:
`UpdateSettingsRequest` for consistency, yes.

> "FBS.Worker.UpdateSettings.Request"

I would not go that way since we would need to create a file type for each request, and it's not more readable than `UpdateSettingsRequest`